sdks/python/setup.py

```diff
@@ -102,6 +102,7 @@ def get_version():
   'google-apitools>=0.5.6,<1.0.0',
   'proto-google-cloud-datastore-v1==0.90.0',
   'googledatastore==7.0.0',
+  'google-cloud-bigquery>=0.22.1,<1.0.0',
```

Comment:
Two questions: 

1. Is it not possible to achieve the same result with the BQ client we have checked in. Otherwise should we be making an effort to move the client out in favor of this library.

2. I would like to avoid an explosion of extra requires for now. Let's just leave a comment here for now and we can investigate more once we have more of such cases as moving them out is fairly easy.